portfolio/src/main/webapp/home.html

```diff
@@ -31,6 +31,10 @@ <h1>Vincent's Portfolio</h1>
         <p>Click here to get a random message:</p>
         <button onclick="addRandomGreeting()">Hello</button>
         <div id="greeting-container"></div>
+        </br>
```

Comment:
It looks like there's something a bit off with the indentation here. All of the tags inside of the <div> that ends on line 38 should be indented one level farther than the div itself